Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -559,6 +559,7 @@ def get_repo_data(regex)
 
       _, user, repo = *regex.match(formula.stable.url) if formula.stable
       _, user, repo = *regex.match(formula.homepage) unless user
+      _, user, repo = *regex.match(formula.head.url) unless user || !formula.head
```

Comment:
```suggestion
      _, user, repo = *regex.match(formula.head.url) if !user && formula.head
```

We generally avoid `unless ... || ...` for readability reasons.

Reply:
Thanks - I changed it to `if ... & ...` per the suggestion.